src/cfront.c

```diff
@@ -1,8 +1,19 @@
 /* C language front-end */
 
+char peek_char(int offset);
```

Comment:
Move the forward declaration `peek_char` right before  the `is_line_continuation`.